tests/snuba/test_organization_discover_saved_query_detail.py

```diff
@@ -0,0 +1,47 @@
+from __future__ import absolute_import
+
+import six
+from sentry.testutils import APITestCase
+from django.core.urlresolvers import reverse
+from sentry.testutils import SnubaTestCase
+
+from sentry.models import DiscoverSavedQuery
+
+
+class OrganizationDiscoverSavedQueryDetailTest(APITestCase, SnubaTestCase):
+    def setUp(self):
+        super(OrganizationDiscoverSavedQueryDetailTest, self).setUp()
+        self.login_as(user=self.user)
+        self.org = self.create_organization(owner=self.user)
+        self.project_ids = [
+            self.create_project(organization=self.org).id,
+            self.create_project(organization=self.org).id
+        ]
+        query = {
+            'fields': ['test'],
+            'conditions': [],
+            'limit': 10
+        }
+
+        model = DiscoverSavedQuery.objects.create(
+            organization=self.org, name="Test query", query=query)
+
+        model.add_projects(self.project_ids)
+
+        self.query_id = model.id
+
+    def test_get(self):
```

Comment:
Do we usually include endpoint tests for permission denied scenarios?

Reply:
Good point, added a basic test for this